extensions/blocks/pinterest/pinterest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Pinterest Block.
+ *
+ * @since 7.x
+ *
+ * @package Jetpack
+ */
+
+jetpack_register_block(
+	'jetpack/pinterest',
+	array( 'render_callback' => 'jetpack_pinterest_block_load_assets' )
+);
+
+/**
+ * Pinterest block registration/dependency declaration.
+ *
+ * @param array  $attr    Array containing the Pinterest block attributes.
+ * @param string $content String containing the Pinterest block content.
+ *
+ * @return string
+ */
+function jetpack_pinterest_block_load_assets( $attr, $content ) {
+	Jetpack_Gutenberg::load_assets_as_required( 'pinterest' );
```

Comment:
You remove this if we don't need CSS file (https://github.com/Automattic/jetpack/pull/13905/files#r341331466).

Would be good place to enqueue `https://assets.pinterest.com/js/pinit.js` tho! Likely good as `<script async defer></script>`

Reply:
👍🏻 2d2b2da15